src/Application/OrganizingEventsService.Application.Models/Dto/Feedback/CreateFeedbackDto.cs

```diff
@@ -1,8 +1,14 @@
+using OrganizingEventsService.Application.Models.CustomValidatorAttributes;
+using System.ComponentModel.DataAnnotations;
+
 namespace OrganizingEventsService.Application.Models.Dto.Feedback;
 
 public class CreateFeedbackDto
 {
-    public uint Rating { get; set; }
+    [Required(ErrorMessage = "Rating is missing")]
+    [DecimalRange(1.0, 10.0, ErrorMessage = "Rating must be between 1.0 and 10.0")]
```

Comment:
```suggestion
    [Range(1, 5, ErrorMessage = "Rating must be between 1 and 5")]
```
Сори, на созвоне неправильно объяснил задачу.